common/lib/protocol-definitions/README.md

```diff
@@ -3,4 +3,19 @@
 Core set of Fluid protocol interfaces shared between services and clients.
 These interfaces should always be back and forward compatible.
 
+**Topics covered below:**
+
+-   [NoOps](#NoOps)
+
+## NoOps
+
+Definition: NoOps are empty operation message, with the type of `MessageType.NoOp`. They are used to send an updated referenceSequenceNumber to service. Relay service is free to coalesce these messages, or fully drop them if another op was used to update Minimum Sequence Number, to a number equal to or higher than the referenceSequenceNumber in Noop.
+
+At the end of a batch of ops, the client will send an NoOp to let the server know it's the end of the batch. Sometimes we can also send NoOp immediately, see both in `CollabWindowTracker.scheduleSequenceNumberUpdate()`
```

Comment:
I'd probably change it and say - client will periodically send noops (and not mentioned batches). I'd definitely try to avoid language that could imply that client sends noops after every batch, as that's not true.

Please note that we no longer send noops for immediate cases, we use MessageType.Accept messages.